communications.php

```diff
@@ -6,6 +6,37 @@
 * Show the latest calls and texts in the $comms variable.
 * 
 */
+
+require_once 'config.php';
+require_once $LIB_BASE . 'lib_sms.php';
+
+// URL parameters
+$start = (int)$_REQUEST['s'];
+$mark = (int)$_REQUEST['mark'];
+$unmark = (int)$_REQUEST['unmark'];
+
+// Settings
+$page = 50;
```

Comment:
This might be unrelatedto what you wrote but this variable confuses me. what is page?